packages/react-native-web/src/exports/Image/index.js

```diff
@@ -146,6 +146,12 @@ function resolveAssetUri(source): ?string {
   return uri;
 }
 
+function hasSourceDiff(a: SourceObject, b: SourceObject) {
+  return (
+    a.uri !== b.uri || JSON.stringify(a.headers) !== JSON.stringify(b.headers)
```

Comment:
If `a.headers` has properties in a different order than `b.headers`, then this will return true even though they could be the exact same, no? That said, in our case we only want to add 1 header at a time so that's not an issue - should we add a comment about that?

Reply:
> if `headers` is build programmatically and starts as a blank `{}`, and then logic adds the headers - it would aways add them in the same order 

True true, I'm down to leave it as is since you have good points, I can't think of a way of this to "break" - and "break" (worst case) only means there's an extra request 🤷 

> So it's possible to drop this logic, but it might result in an active request (no cache yet) getting aborted and restarted

Agreed let's keep it for now 👍